common/app/srs.js

```diff
@@ -0,0 +1,72 @@
+import * as R from 'ramda'
```

Comment:
in common you need to user require('ramda'). es6 imports do not work in node

Reply:
done